src/context/GameContext.js

```diff
@@ -0,0 +1,202 @@
+import { useState, createContext } from 'react';
+
+const UserContext = createContext();
```

Comment:
Generally, you want to name your context so that it makes it clear what the context is for. Is there a reason that you are calling this `UserContext? `?

Reply:
nope, must of been a brainfart because I even used GameContext.js as the file name LOL